composer.json

```diff
@@ -21,7 +21,8 @@
     "require": {
         "php": "^7.4 | ^8.0",
         "ext-json": "*",
-        "symfony/event-dispatcher": "~4.0 | ~5.0 | ~6.0"
+        "symfony/event-dispatcher": "~4.0 | ~5.0 | ~6.0",
+        "ext-mongodb": "^1.12"
```

Comment:
1. Better use `*` for version extension.
2. It's not primary required for any project. Must be added only to `require-dev`.
3. Please sort all required, first - extension, second - other packages.